src/bundle/Resources/views/admin/content_type/edit.html.twig

```diff
@@ -44,7 +44,7 @@
     <section class="container mt-4 px-5">
         <div class="card ez-card">
             <div class="ez-card__header ez-card__header--secondary d-flex justify-content-between">
-                <div class="p-2">
+                <div class="ez-card__title p-2">
```

Comment:
Still, why do we need this extra class? It has the same styling as `ez-card__header--secondary` (color and font-weight), this styling will be inherited from the parent.

Reply:
Update it based on review 👍